### DISCUSSION TITLE
Can you generate patients encounters/hospital visit records for a specific 2 year period?

### DISCUSSION THREAD

**Discussion by r0sies** [synthetichealth/synthea#1274] [Q&A] [answered]
When I generate a population, patient encounters/hospital admissions span 100+ years. Is it possible to generate a population where there are thousands of encounters/hospital admissions for a specific 2 year period? or is it a matter of generating a very large population and hope you get a large enough sample over the 2-year period of interest. Thanks!

**Comment by hadleynet** [comment, marked as answer]
You can run Synthea with the following switches: `--exporter.years_of_history=2 --generate.only_alive_patients=true`. This will ensure you only have the same final two years of history for each patient.
